.github/workflows/gh-ci-cron.yaml

```
@@ -269,7 +272,7 @@ jobs:
     strategy:
         fail-fast: false
         matrix:
-          os: [ubuntu, macos, windows]
+          os: [ubuntu, macos, macos-14, windows]
```

@RMeli the reason this isn't running is beacuse it's doing "${os}-latest", so it's trying "macos-14-latest".

The short term fix is just to remove `latest` on 270 and then do:

```suggestion
          os: [ubuntu-latest, macos-latest, macos-14, windows-latest]
```